### PR TITLE
Adds basic ORCA docs

### PR DESCRIPTION
**Pull request by aturner-epcc**

### CODE REVIEW

File: docs/research-software/orca.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    support for the software and you may be directed to seek support from
+    other places if the service desk cannot answer the questions.
+
+## Useful Links
+
+- [ORCA Forum](https://orcaforum.kofo.mpg.de/app.php/portal)
+
+## Using ORCA on ARCHER2
+
+ORCA s available for academic use on ARCHER2 only. If you wish to use ORCA for commercial
```
Comment (juanfrh) [inline docs/research-software/orca.md:19]: Typo: ORCA **is** available [...]

**Review by juanfrh — COMMENTED** [review]
I only found a typo. The rest looks good to me. Thanks Andy.

File: docs/research-software/orca.md (hunk 1 above)
Comment (aturner-epcc) [reply docs/research-software/orca.md]: Well spotted @juanfrh , fixed now.

**Review by juanfrh — APPROVED** [review]
LGTM

**Pull request merged**